onsite/phospho_scoring.py

```
@@ -0,0 +1,643 @@
+
+            results.append({
+                'new_sequence': new_seq_str,
+                'best_ascore': best_ascore,
+                'meta_fields': meta_fields
+            })
+
+        return {'status': 'success', 'hits': results}
+    except Exception as e:
+        return {'status': 'error', 'reason': str(e)}
+
+def find_spectrum_by_mz(exp, target_mz, rt=None, ppm_tolerance=10):
```

Ruff (0.14.0)

Unused function argument: `rt`

(ARG001)